src/go/k8s/tests/e2e/console-admin-api/00-assert.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: console-admin-api
```

Comment:
You don't need to create static namespace. Kuttl tests harness handle that.

Reply:
I think there is no way to reference that kuttl autogenerated namespace in the Console ClusterRef